frontend/src/components/StableCoin/Holders.vue

```diff
@@ -58,38 +49,63 @@
 	</div>
 </template>
 <script lang="ts" setup>
-import { numberFormatter } from '~/utils/format'
-import { useStableCoinDashboardList } from '~/queries/useStableCoinDashboardList'
 import FtbCarousel from '~/components/molecules/FtbCarousel.vue'
 import CarouselSlide from '~/components/molecules/CarouselSlide.vue'
 import BWCubeLogoIcon from '~/components/icons/BWCubeLogoIcon.vue'
 import Filter from '~/components/StableCoin/Filter.vue'
+import { usePltTokenHolderQuery } from '~/queries/usePltTokenHolderQuery'
+import type { PltAccountAmount } from '~/types/generated'
+import { usePagedData } from '~/composables/usePagedData'
+
 import {
+	holderFilterOptions,
 	TransactionFilterOption,
-	transactionFilterOptions,
 } from '~/types/stable-coin'
+import { ref, watch } from 'vue'
 
 // Define Props
 const props = defineProps<{
-	coinId?: string
+	coinId: string
+	totalSupply?: bigint
```

Comment:
Can the `totalSupply` be undefined? Isn't the database always having a value as there is a `NOT NULL` constraint on the database column.